app/src/main/java/org/simple/clinic/medicalhistory/newentry/NewMedicalHistoryModel.kt

```diff
@@ -56,6 +57,12 @@ data class NewMedicalHistoryModel(
   val showOngoingDiabetesTreatment: Boolean
     get() = diagnosedWithDiabetes && country.isoCountryCode == Country.INDIA
 
+  val hasNoHypertension: Boolean
+    get() = ongoingMedicalHistoryEntry.diagnosedWithHypertension == No
+
+  val hasNoDiabetes: Boolean
```

Comment:
We're only using `hasNoHypertension` and `hasNoDiabetes` in the model, can we make this private

Reply:
Ah, good catch. I moved them to resolve to code climate issue and forgot to change visibility.